indexer/services/ender/src/config.ts

```diff
@@ -29,6 +29,9 @@ export const configSchema = {
   USE_SUBACCOUNT_UPDATE_SQL_FUNCTION: parseBoolean({
     default: true,
   }),
+  USE_SQL_FUNCTION_TO_CREATE_INITIAL_ROWS: parseBoolean({
+    default: true,
+  }),
```

Comment:
I feel good always using SQL to create inital rows, less logic for us to keep track of too!

Reply:
I'd like to keep this as is for now. It'll be easy to turn off in case there are any issues.
We can deprecate the old logic in the future.